md/BonitaBPM_platform_setup.md

```diff
@@ -137,10 +139,40 @@ setup.bat configure "-Ddb.vendor=postgres" "-Ddb.server.name=localhost" "-Ddb.se
 
 ::: warning
 For Windows users: Due to Windows Batch limitations, only 8 parameters are supported.
-If you need to pass more than 8 parameters, use file `database.properties` instead.
+If you need to pass more than 8 parameters, modify file `database.properties` instead.
 :::
 
 
+### Advanced database configuration using file internal.properties
+
+The file `internal.properties` is used internally by the setup tool to properly configure and initialize the bundle.
+It is made of both data entered in file `database.properties` as well as other data like database driver class name, connection URL, etc.  
+This file **should not** be modified manually in most cases, unless for specific use-cases like adding parameters in the connection URL or using a specific database driver.
+
+This file contains the Database configuration information that are not inside file `database.properties` (database driver class name, connection URL, etc).
+Those information are used internally by the tool to configure properly the bundle (See [configure command](#run_bundle_configure)) and the database initialization procedure (See [init command](#init_platform_conf)).  
+The setup tool uses the values provided in file `database.properties` as replacement strings to the properties defined in file `internal.properties`. Those new processed values are then used by the tool.
+
+_Usage_:  
+You are allowed to modify these values if, for instance, for Oracle RAC, you need to add parameters in the **connection URL**, or for mysql you need to add characterEncoding or other parameters: 
+
+```properties
+   oracle.url=jdbc:oracle:thin:@(description=(address_list=(address=(protocol=tcp)(port=${db.server.port})(host=${db.server.name})))(connect_data=(INSTANCE_NAME=${db.database.name}))(source_route=yes))
```

Comment:
since oracle RAC is support to have several node, we may use this pattern in example

Reply:
can you tell me what it is?

Reply:
basic example with 2 servers: 
```

myrac.us.oracle.com =
  (DESCRIPTION = 
    (ADDRESS = (PROTOCOL = TCP)(HOST = myrac1.us.oracle.com)(PORT = 1521))
    (ADDRESS = (PROTOCOL = TCP)(HOST = myrac2.us.oracle.com)(PORT = 1521))
    (LOAD_BALANCE = ON)
    (FAILOVER = OFF)
    (CONNECT_DATA =
      (SERVICE_NAME = myrc.us.oracle.com)
      (FAILOVER_MODE = (TYPE=SELECT)(METHOD=BASIC))
    )
  )
```